### PR TITLE
Fix alg2

### PR DESCRIPTION
**Pull request by MaxBlesch**
### What problem do you want to solve?

gettsim produced false values because of the check of benefits(kinderzuschlag, wohngeld and arbeoitsl_geld_2) against the incomes on the individual level. We want to calculate all these things on the household level.

### CODE REVIEW

File: gettsim/benefits/arbeitsl_geld_2/arbeitsl_geld_2.py [hunk 1]
```diff
@@ -1,26 +1,43 @@
...
+    wohngeld_kinderzuschlag_vorrang_hh,
+    anz_rentner_hh,
+):
+    out = arbeitsl_geld_2_m_minus_eink_hh.clip(lower=0)
+    cond = (
+        wohngeld_vorrang_hh
+        | kinderzuschlag_vorrang_hh
+        | wohngeld_kinderzuschlag_vorrang_hh
+    )
+    out.loc[cond | (anz_rentner_hh != 0)] = 0
```
Comment (tobiasraabe) [inline gettsim/benefits/arbeitsl_geld_2/arbeitsl_geld_2.py:17]:
```suggestion
        | (anz_rentner_hh != 0)
    )
    out.loc[cond] = 0
```

File: gettsim/benefits/benefit_checks/vermoegens_checks.py [hunk 2]
```diff
@@ -32,15 +32,18 @@ def _wohngeld_basis_hh_vermögens_check(
     TODO: Need to write numbers to params.
```
Comment (tobiasraabe) [inline gettsim/benefits/benefit_checks/vermoegens_checks.py:32]: Do it now?

File: gettsim/benefits/wohngeld.py [hunk 3]
```diff
@@ -1,6 +1,17 @@
 import numpy as np
 
 
+def wohngeld_m_hh(
+    wohngeld_vermögens_check_hh,
+    wohngeld_vorrang_hh,
+    wohngeld_kinderzuschlag_vorrang_hh,
+    anz_rentner_hh,
+):
+    cond = ~wohngeld_vorrang_hh & ~wohngeld_kinderzuschlag_vorrang_hh
```
Comment (tobiasraabe) [inline gettsim/benefits/wohngeld.py:10]: one condition.

File: gettsim/benefits/arbeitsl_geld_2/arbeitsl_geld_2.py (hunk 1 above)
Comment (MaxBlesch) [reply gettsim/benefits/arbeitsl_geld_2/arbeitsl_geld_2.py]: Done!

File: gettsim/benefits/wohngeld.py (hunk 3 above)
Comment (MaxBlesch) [reply gettsim/benefits/wohngeld.py]: Done!

File: gettsim/benefits/benefit_checks/vermoegens_checks.py (hunk 2 above)
Comment (MaxBlesch) [reply gettsim/benefits/benefit_checks/vermoegens_checks.py:32]: Please not. I am really done for today. Just want that it works and then I am out!

Comment (MaxBlesch) [reply gettsim/benefits/benefit_checks/vermoegens_checks.py:32]: Created #207

**Comment by hmgaudecker** [comment]
Great work here! I think we are ready to release 0.3.2 now.

**Pull request merged**